layouts/default.vue

```diff
@@ -170,6 +204,41 @@ export default {
       }
       return layer
     },
+    getFeatureInfo(bbox) {
+      if (!this.getActiveRasterLayer) {
+        this.removeInfoText()
+        return
+      }
+      const parameters = {
+        imageId: this.activeRasterData.imageId,
+        bbox: bbox,
+        band: bands[this.getActiveRasterLayer],
+      }
+      fetch(`${process.env.HYDRO_ENGINE}/get_feature_info`, {
```

Comment:
This is too hidden, move it up in the code.

Reply:
Don't forget to update the `.env.example` and to update the deployment configs.

Reply:
The url should come from backend.

Reply:
fixed in travis and .env